.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,45 @@
+name: prerelease
```

Comment:
I'm not sure this workflow will actually do anything. It's targeting the root directory, not the packages themselves. We've never updated the Next.js one to work with a monorepo, which is presumably where you got this? I'd remove it

Reply:
Will do.